src/annotations.cr

```diff
@@ -27,7 +27,7 @@ end
 annotation Flags
 end
 
-# A `lib` can be marked with `@[Link(lib : String, ldflags : String, static : Bool, framework : String)]`
+# A `lib` can be marked with `@[Link(lib : String, *, ldflags : String, static : Bool, framework : String, pkg_config : String)]`
```

Comment:
You should add that `static` is deprecated. Not in this line but further down. And maybe not mention it here at all.

Reply:
Should also note that `CRYSTAL_LIBRARY_PATH` is added to the lookup paths.

Reply:
I tweaked the link annotation docs. `CRYSTAL_LIBRARY_PATH` was already mentioned further down, but I removed some old references to hardcoded paths.

I removed mention of `static` entirely, I presume that'll be gone by 1.0 if all goes well.